internetarchive/bookreader
# Es6 plugin: Search

#### Test Steps
- pull branch
- fresh module install, cmd: `npm install`
- run server: `npm run serve`
- Test desktop
  - go to: http://127.0.0.1:8000/BookReaderDemo/demo-ia-olivertwist.html
  - search text using input: `child` (wait a while, then wait a lil more) -->  results markers successfully pop up
  - hover over results marker. --> text tooltip pops up
    - known: page highlight bleeds over tooltip
  - hover off of results marker --> tooltip disappears
  - click on marker --> takes you to page with results highlighted
  - refresh page that is full of search results (wait for it...) --> results markers show
  - search a new word: `shmeowza` --> no results window pops up & fades out
- Run tests on mobile device (browser dev emulator)
  - go to: http://127.0.0.1:8000/BookReaderDemo/demo-ia-olivertwist.html
  - open 🍔 --> search section is visible
 - search text using input: `child` (wait a while, then wait a lil more) -->  results & count appear below input
  - click on search result -> takes you to page
 - refresh page --> results are shown in the search drawer
  - search a new word: `shmeowza` --> no results are displayed


internal ticket: https://webarchive.jira.com/browse/WEBDEV-3206
